src/App.tsx

```diff
@@ -1,11 +1,13 @@
-import React from "react";
+import React from 'react';
+import Speakers from './Components/Speakers/Speakers';
 
 function App() {
```

Comment:
We can use {useState, useEffect} in the App function to have a loading timer so to render it completely. 

```
function App() {

  const [loading, setLoading] = useState(true);
  useEffect(() => {
    setTimeout(() => {
      setLoading(false);
    }, 3500);
  }, []);

    return (
        <div>
            {/* <h1 className='text-3xl font-bold underline'>Cloud Community Days</h1> */}
            {loading ? 
            <Loading/>
            :
            <Speakers />}  
        </div>
    );
}
```

Reply:
Thanks cosmos